pkg/network/node/sdn_controller.go

```diff
@@ -111,6 +111,13 @@ func (plugin *OsdnNode) SetupSDN() (bool, map[string]podNetworkInfo, error) {
 		return false, nil, fmt.Errorf("net/ipv4/ip_forward=0, it must be set to 1")
 	}
 
+	// In OCP 4 apparently we have a lot of rejects.
+	// Default rate mask is 6188, 6160 is default without destination unreachable
+	// FIXME: Why is this new to OCP 4?
```

Comment:
It's not. We skipped the "should be rejected" test in 3.x, then it seemed to be working in 4.x so we re-enabled it, except then it turned out to still be flaky for reasons unknown.

Reply:
Same symptoms but not exactly the same cause, the 3.x bug was happening because we didn't have an iptables rules that would reject *forwarded* packets, however, if you trid to reach a service without endpoints from the host network it was rejected.
More https://bugzilla.redhat.com/show_bug.cgi?id=1782857